understanding/20/images-of-text.html

```diff
@@ -44,6 +44,11 @@ <h2>Intent of Images of Text</h2>
       <p>Images of text can also be used where it is possible for users to customize the image
          of text to match their requirements.
       </p>
+
+      <p>If it is not possible to replace an image of text with actual text or to allow users to customize the image, the author must provide an [accessible alternative](https://www.w3.org/TR/WCAG22/#cc2) by duplicating the meaning or content of the image in text form. 
+         If the information within the image permits the author to replicate the text verbatim, this should be the preferred approach. 
+         However, if converting the visual image to text results in a loss of meaning or readability, it is acceptable to convey the same message by adding or altering some words.
```

Comment:
```suggestion
         The alternative could replicate the text verbatim. If converting the visual image to text results in a loss of meaning or readability, it could add or altering some words to convey the style, emphasis, or whatever meaning was lost.
```

Reply:
`it could add or altering` > `it could add or alter` or even `it could add or change`